client/jetpack-cloud/sections/partner-portal/license-preview/index.tsx

```diff
@@ -50,7 +50,7 @@ export default function LicensePreview( {
 		domain && [ LicenseState.Attached, LicenseState.Revoked ].indexOf( licenseState ) !== -1;
 	const justIssued =
 		moment.utc( issuedAt, 'YYYY-MM-DD HH:mm:ss' ) > moment.utc().subtract( 1, 'minute' );
-
+	const justAssigned = justIssued && 'attached' === licenseState;
```

Comment:
- Let's use the enum we already have rather: LicenseState.Attached
- justIssued cannot be reused here as a license can be assigned days after it has been issued so we should adjust the behavior to match but test the attachedAt date rather than the issuedAt date:
```
const aMinuteAgo = moment.utc().subtract( 1, 'minute' );
const justIssued = moment.utc( issuedAt, 'YYYY-MM-DD HH:mm:ss' ) > aMinuteAgo;
const justAssigned =
    moment.utc( attachedAt, 'YYYY-MM-DD HH:mm:ss' ) > aMinuteAgo &&
    LicenseState.Attached === licenseState;
```